search/controller/weedid/views.py

```diff
@@ -0,0 +1,18 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+import requests
+
+
+def test(request):
+    return HttpResponse("TESTING VIEW")
+
+
+@csrf_exempt
+def elasticsearchQuery(request):
+    elasticsearch_url = "/".join(request.path.split("/")[3:])
```

Comment:
We'll need to make sure that this is either using a read-only account or restricts access to read-only URLs.